astropy/io/ascii/latex.py

```diff
@@ -228,6 +228,11 @@ class Latex(core.BaseReader):
     This class can also read simple LaTeX tables (one line per table
     row, no ``\multicolumn`` or similar constructs), specifically, it
     can read the tables that it writes.
+    When reading, it will look for LaTeX command to start a tabular data
```

Comment:
Slight edit (missing "the", data is plural; purpose of what?):
```
When reading, it will look for the Latex commands to start and end tabular
data (``\begin{tabular}`` and ``\end{tabular}``). That means that
those lines have to be present in the input file; the benefit is that this
```

```